docs/nodes/compute/advanced/local-testing.md

```diff
@@ -0,0 +1,99 @@
+# Running Instances on the Local Environment
+
+Aleph.im supports two popular and high-performance hypervisors for virtualization: Firecracker and Qemu. This guide outlines the steps to run instances on your local machine for development and testing purposes.
+
+## 0. Prerequisites
+
+- **SSH Keys:** Ensure you have a valid SSH key pair set up. You'll need the public key to enable SSH access to the instances.
+
+## 1. Configuration
+
+### **Configure Allocation Token and SSH Keys:**
+
+- Edit the `/etc/aleph/supervisor.env` file and add the following lines:
+
+  ``` bash
+  # Encrypted allocation token hash in sha1 format
+  ALEPH_VM_ALLOCATION_TOKEN_HASH=YOUR_ALLOCATION_TOKEN_HASH
+
+  # List of authorized SSH public keys (replace "YOUR_SSH_PUBLIC_KEY" with your actual key)
+  ALEPH_VM_DEVELOPER_SSH_KEYS=["YOUR_SSH_PUBLIC_KEY"]
+  ```
+
+- **Important:** Replace `YOUR_ALLOCATION_TOKEN_HASH` with your actual encrypted allocation token and `YOUR_SSH_PUBLIC_KEY` with your public SSH key.
+
+
+### **Enable Developer SSH Keys (Optional):**
+
+By default, SSH access to instances is only enabled to the user that creates the VM for security reasons. To enable it for development, you have two options:
+
+#### a) Modifying the Systemd Service File:
+
+1. Locate the Aleph VM orchestrator service file, typically found at `/etc/systemd/system/aleph-vm-supervisor.service`.
+2. Edit the file and add the `--developer-ssh-keys` argument to the `ExecStart` line. It should look something like this:
+
+   ``` bash
+   ExecStart=python3 -m aleph.vm.orchestrator --print-settings --very-verbose --developer-ssh-keys
+   ```
+
+3. Reload the systemd configuration and restart the service:
+
+   ``` bash
+   sudo systemctl daemon-reload
+   sudo systemctl restart aleph-vm-supervisor
+   ```
+
+#### b) Running the Orchestrator Manually:
+
+You can simply append the `--developer-ssh-keys` argument when running the orchestrator manually:
+
+ ``` bash
+ python3 -m aleph.vm.orchestrator --print-settings --very-verbose --developer-ssh-keys
+ ```
+
+## 2. Launch Instance
+
+Use the following `curl` command to launch an instance locally:
+
+ ``` bash
```

Comment:
Should we provide an actual working ItemHash for the example ?
What about assigning it in a variable and then using it in the `curl` command ?

Reply:
This isn't a script, it is just the documentation that shows how to run an instance, so deal with variables or something like this can be difficult to understand to users.
About the default Item hash, if we have 2 test Item hashes to use, we can include it in the docs, sure.